sap-file-server-md/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "SAP NetWeaver file server - MD",

```

Comment:
MD might be ambiguous to some folks - suggest: "SAP NetWeaver File Server (Managed Disks)"